sbnanaobj/StandardRecord/SRCaloPoint.h

```diff
@@ -0,0 +1,44 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRCaloPoint.h
+////////////////////////////////////////////////////////////////////////
+#ifndef SRCALOPOINT_H
+#define SRCALOPOINT_H
+
+#include "sbnanaobj/StandardRecord/SRTrueCaloPoint.h"
+
+namespace caf
+{
+
+  // Calo information on a point
+  class SRCaloPoint {
+    public:
+
+      SRCaloPoint();
+      virtual ~SRCaloPoint() {}
```

Comment:
Destructor worse than useless. I know this is a pattern seen in other standard record classes... but nowhere it makes sense.
I recommend obliteration.

Reply:
Fixed.